src/components/ImgFromId/ImgFromIdContainer.js

```diff
@@ -7,7 +7,7 @@ import { getMediaPath } from 'common/utils/mediaHelpers'
 import { IMAGE, MEDIUM } from 'common/constants'
 
 function ImgFromIdContainer(props) {
-  const { id, size, alt, className, ...other } = props
+  const { mockData, id, size, alt, className, ...other } = props
   const [src, setSrc] = useState('')
 
```

Comment:
This is a weird enough way of using this component that I think an explanatory comment is in order. (It's called ImageFromId but in this case you are passing a src path in the id parameter instead.)

If we keep the homepage this way I think we should refactor this to have better naming, but for now I think it's ok